src/convertToObject.js

```diff
@@ -5,8 +5,21 @@
  *
  * @return {object}
  */
-function convertToObject(sourceString) {
-  // write your code here
-}
+const convertToObject = (sourceString) => {
+  const splittedSource = sourceString.split(';');
+  const resultObj = {};
+
+  for (const element of splittedSource) {
+    const resultArray = element.replace(/[[\n\t]/g, '').split(':');
+
+    if (resultArray.length === 2) {
+      resultObj[resultArray[0].trim()] = resultArray[1]
+        .replace(/[',']/g, ',\n')
+        .trim();
+    }
+  }
```

Comment:
Instead of using a for loop, consider using array iteration methods such as 'map' and 'reduce' to transform the array of style declarations into an object. This would be more in line with functional programming practices and the checklist item 5.